### PR TITLE
fix(config): marketplace uploadDocumentTypeIds

### PR DESCRIPTION
**Pull request by ntruchsess**
## Description

add uploadDocumentTypeIds setting to marketplace config

## Why

The app-marketplace settings uploadDocumentTypeIds scheme has been changed from generic dictionary-type to strongly types the previous configuration rendering the existing configuration invalid. As this is a mandatory config the service app-marketplace fails to start

## Issue

N/A (CPLP-2891)

## Link to pull request from other repository.

https://github.com/eclipse-tractusx/portal-backend/pull/90

## Checklist

Please delete options that are not relevant.

- [X] I have performed a self-review of my changes
- [X] I have successfully tested my changes
- [X] I have added comments in the default values.yaml file with helm-docs syntax ('# -- ') if relevant for installation

### CODE REVIEW

**Review by Phil91 — APPROVED** [review]

File: charts/portal/templates/deployment-backend-services.yaml [hunk 1]
```diff
@@ -148,18 +148,18 @@ spec:
...
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting0 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__DOCUMENTTYPEID"
-          value: "{{ .Values.backend.appmarketplace.type2 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__0"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting1 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__1"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting2 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__2"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting3 }}"
+        - name: "SERVICES__UPLOADAPPDOCUMENTTYPEIDS__0__DOCUMENTTYPEID"
```
Comment (evegufy) [inline charts/portal/templates/deployment-backend-services.yaml:151]: It needs to be UPLOADSERVICEDOCUMENTTYPEIDS in the services services.
The helm test workflow failed due to that:
https://github.com/eclipse-tractusx/portal-cd/actions/runs/5327839719/jobs/9651769407?pr=30#step:9:4397

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/portal/templates/deployment-backend-services.yaml (hunk 1 above)
Comment (ntruchsess) [reply charts/portal/templates/deployment-backend-services.yaml]: fixed

File: charts/portal/templates/deployment-backend-services.yaml [hunk 2]
```diff
@@ -149,17 +149,17 @@ spec:
         - name: "SERVICES__CATENAADMINROLES__0__USERROLENAMES__0"
           value: "{{ .Values.backend.services.catenaAdminRoles.role0 }}"
         - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__0__DOCUMENTTYPEID"
-          value: "{{ .Values.backend.appmarketplace.type0 }}"
+          value: "{{ .Values.backend.appmarketplace.uploadAppDocumentTypeIds.documentTypeId0 }}"
```
Comment (evegufy) [inline charts/portal/templates/deployment-backend-services.yaml:152]: Please refer to services.uploadServiceDocumentTypeIds for the services service
https://github.com/eclipse-tractusx/portal-cd/blob/8f1b6f67e76c38f9d96e2b6932d12f50a55660e3/charts/portal/values.yaml#L594

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/portal/templates/deployment-backend-services.yaml (hunk 2 above)
Comment (ntruchsess) [reply charts/portal/templates/deployment-backend-services.yaml]: fixed

**Review by evegufy — APPROVED** [review]

**Pull request merged**